monitoring_service.rst

```diff
@@ -2,145 +2,230 @@ Raiden Network Monitoring Service
-
-Appendix B: Message format
+Appendix B: Message Format
 ==========================
 Monitoring service uses JSON format to exchange the data.
 For description of the envelope format and required fields of the message please see Transport document.
 
+Monitor Request
+---------------
+Monitor Requests are generated by clients from `Onchain Balance Proofs`__.
 
-Balance proof
--------------
-* nonce (uint64) - it is expected that nonce is incremented by 1 with each Balance Proof exchanged between Channel Participants
-* transferred_amount (uint256) - amount of tokens transferred
-* channel_address (address) - address of the netting channel
-* locksroot (bytes32) - lock root state of the channel
-* extra_hash (bytes32) - implementation dependent extra data
-* signature (bytes32) - ecrecoverable signature of the data above, in order they are listed here
++--------------------------+------------+--------------------------------------------------------------------------------+
+| Field Name               | Field Type |  Description                                                                   |
++==========================+============+================================================================================+
+|  onchain_balance_proof   | object     | Onchain Balance Proof                                                          |
++--------------------------+------------+--------------------------------------------------------------------------------+
+|  non_closing_signature   | string     | Signature of the Onchain Balance Proof by the client                           |
++--------------------------+------------+--------------------------------------------------------------------------------+
+|  reward_amount           | uint256    | Proposed fee in RDN                                                            |
++--------------------------+------------+--------------------------------------------------------------------------------+
+|  reward_proof_signature  | string     | Signature of parts of the Onchain Balance Proof - channel_identifier,          |
+|                          |            | token_network_address, chain_id, nonce - and the reward_amount                 |
++--------------------------+------------+--------------------------------------------------------------------------------+
+
+__ https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#balance-proof-onchain
 
-All of this fields are required. Monitoring Service MUST perform basic verification of these data, namely channel existence. Monitoring service SHOULD accept the message if and only the sender of the message is same as the sender address recovered from the signature.
+All of this fields are required. Monitoring Service MUST perform basic verification of these data, namely channel
```

Comment:
what is *basic* verification? I'd just remove that word.

Reply:
good point